yurt/orchestration/roles/db/tasks/main.yml

```diff
@@ -46,14 +46,26 @@
                    role_attr_flags=NOSUPERUSER,NOCREATEDB
                    state=present
 
+- name: Create Client Authentication file if it does not exist yet
```

Comment:
Yeah. Default is `force:yes` which means if the file is at all different, it overwrites. `force:no` means it doesn't overwrite, even if there's file differences.